<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="openGlWidget" name="glWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>85</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Charger un fichier&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Fichier</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSymmetry">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>85</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Symmetry</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointGroupTitle">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>400</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Point Group</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointGroupLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C&lt;sub&gt;1&lt;/sub&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>350</y>
      <width>150</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Chemical Formula</string>
    </property>
   </widget>
   <widget class="QLabel" name="chemicalFormulaLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>450</y>
      <width>150</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Molar mass</string>
    </property>
   </widget>
   <widget class="QLabel" name="molarMassLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonColor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>85</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSymmetrize">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>85</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Symmetrize</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>openGlWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>src/openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
